packages/ibm-products/src/components/Datagrid/Datagrid/DatagridToolbar.tsx

```diff
@@ -5,11 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useRef, MutableRefObject, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import {
+  /**@ts-ignore */
```

Comment:
Rather than adding `ts-ignore`s to these few lines, can you add these components to the list in our `custom-typings/index.d.ts` declaration file from `@carbon/react`.

Reply:
https://github.com/carbon-design-system/ibm-products/blob/main/packages/ibm-products/src/custom-typings/index.d.ts